// hibp/lib/hibp.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.search = exports.pasteAccount = exports.dataClasses = exports.breaches = exports.breachedAccount = exports.breach = undefined;

var _breach = require('./breach');

var _breach2 = _interopRequireDefault(_breach);

var _breachedAccount = require('./breachedAccount');

var _breachedAccount2 = _interopRequireDefault(_breachedAccount);

var _breaches = require('./breaches');

var _breaches2 = _interopRequireDefault(_breaches);

var _dataClasses = require('./dataClasses');

var _dataClasses2 = _interopRequireDefault(_dataClasses);

var _pasteAccount = require('./pasteAccount');

var _pasteAccount2 = _interopRequireDefault(_pasteAccount);

var _search = require('./search');

var _search2 = _interopRequireDefault(_search);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

/*
 * Export individual named functions in an anonymous object to allow the
 * following (and support tree-shaking):
 *
 * import * as hibp from 'hibp';       // ESM (with tree-shaking)
 * import { search } from 'hibp';      // ESM (with tree-shaking)
 * const { search } = require('hibp'); // CommonJS
 * const hibp = require('hibp');       // CommonJS
 */
/**
 * An object/namespace containing all the hibp functions.
 *
 * @namespace hibp
 * @example
 * import * as hibp from 'hibp';
 * // Now all hibp functions are available on the hibp object:
 * hibp.dataClasses().then(...)
 */
exports.breach = _breach2.default;
exports.breachedAccount = _breachedAccount2.default;
exports.breaches = _breaches2.default;
exports.dataClasses = _dataClasses2.default;
exports.pasteAccount = _pasteAccount2.default;
exports.search = _search2.default;
